server/live-loan-router.js

```diff
@@ -0,0 +1,125 @@
+const get = require('lodash/get');
+const express = require('express');
+const argv = require('./util/argv');
+const config = require('../config/selectConfig')(argv.config);
+const fetch = require('./util/fetch');
+
+const drawLoanCard = require('./util/live-loan/live-loan-draw');
+
+function getRecommendedLoans(type, loginId, offset = 0, limit = 1) {
+	return new Promise((resolve, reject) => {
+		const endpoint = config.app.graphqlUri;
+		let query;
+		if (type === 'url') {
+			query = `{
+				ml {
+					recommendationsByLoginId(
+						segment: all
+						  loginId: ${loginId}
+						  offset: ${offset}
```

Comment:
Note, I have not been able to make any offset other than 0 work in the graphQl query. Will check in with the Ml team.

Reply:
You could also consider just fetching like 3 loans, then filtering and serving accordingly

Reply:
Ml team is working on offset. Faking it for now using by setting limit to the offset number, then using the last loan from the result.

Reply:
Do we even need to use the offset at all in this query? Couldn't we always fetch 4 loans, cache that result, and then use the offset argument to pick which of those loans in the cache to return? Fundraising loans are already going to be filtered out by the ML endpoint, so we shouldn't be in a situation where we need to fetch more than the initial amount of loans.

Reply:
> Do we even need to use the offset at all in this query? Couldn't we always fetch 4 loans, cache that result, and then use the offset argument to pick which of those loans in the cache to return? Fundraising loans are already going to be filtered out by the ML endpoint, so we shouldn't be in a situation where we need to fetch more than the initial amount of loans.

Love it, now fetching 4 loans and caching the result to memcache for 30 minutes, then using offset to grab the right entry.